sockeye/pre_mxnet.py

```diff
@@ -34,6 +35,12 @@ def handle_omp_num_threads():
                 val = sys.argv[i + 1]
             logger.warning('Setting %s=%s', OMP_NUM_THREADS, val)
             os.environ[OMP_NUM_THREADS] = val
+        elif arg == ENV_ARG and len(sys.argv) >= i + 1:
+            argval = sys.argv[i + 1]
+            for var_val in argval.split(';'):
```

Comment:
Minor comment 2: Using comma as a separator would be more consistent with other args like custom optimizer params.

Reply:
makes sense, will change from ';' to ','.